docs/release-1.2/existing-rhdh.md

```diff
@@ -64,121 +91,156 @@ metadata:
+
+To incorporate the Orchestrator plugins, append the following configuration to the **dynamic-plugins** ConfigMap:
 
 ```yaml
-  - disabled: false
-    package: "@redhat/backstage-plugin-orchestrator-backend-dynamic@1.2.0"
-    integrity: sha512-lyw7IHuXsakTa5Pok8S2GK0imqrmXe3z+TcL7eB2sJYFqQPkCP5la1vqteL9/1EaI5eI6nKZ60WVRkPEldKBTg==
-    pluginConfig:
-      orchestrator:
-        dataIndexService:
-          url: http://sonataflow-platform-data-index-service.sonataflow-infra
-  - disabled: false
-    package: "@redhat/backstage-plugin-orchestrator@1.2.0"
-    integrity: sha512-FhM13wVXjjF39syowc4RnMC/gKm4TRlmh8lBrMwPXAw1VzgIADI8H6WVEs837poVX/tYSqj2WhehwzFqU6PuhA==
-    pluginConfig:
-      dynamicPlugins:
-        frontend:
-          janus-idp.backstage-plugin-orchestrator:
-            appIcons:
-              - importName: OrchestratorIcon
-                module: OrchestratorPlugin
-                name: orchestratorIcon
-            dynamicRoutes:
-              - importName: OrchestratorPage
-                menuItem:
-                  icon: orchestratorIcon
-                  text: Orchestrator
-                module: OrchestratorPlugin
-                path: /orchestrator
+      - disabled: false
+        package: "@redhat/backstage-plugin-orchestrator-backend-dynamic@1.2.0"
+        integrity: sha512-lyw7IHuXsakTa5Pok8S2GK0imqrmXe3z+TcL7eB2sJYFqQPkCP5la1vqteL9/1EaI5eI6nKZ60WVRkPEldKBTg==
+        pluginConfig:
+          orchestrator:
+            dataIndexService:
+              url: http://sonataflow-platform-data-index-service.sonataflow-infra
+      - disabled: false
+        package: "@redhat/backstage-plugin-orchestrator@1.2.0"
+        integrity: sha512-FhM13wVXjjF39syowc4RnMC/gKm4TRlmh8lBrMwPXAw1VzgIADI8H6WVEs837poVX/tYSqj2WhehwzFqU6PuhA==
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              janus-idp.backstage-plugin-orchestrator:
+                appIcons:
+                  - importName: OrchestratorIcon
+                    module: OrchestratorPlugin
+                    name: orchestratorIcon
+                dynamicRoutes:
+                  - importName: OrchestratorPage
+                    menuItem:
+                      icon: orchestratorIcon
+                      text: Orchestrator
+                    module: OrchestratorPlugin
+                    path: /orchestrator
 ```
 
-The versions of the plugins may undergo updates, leading to changes in their integrity values. To ensure you are utilizing the latest versions, please consult the values available [here](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/values.yaml). It's imperative to set both the version and integrity values accordingly.
+Additionally, ensure that the `dataIndexService.url` in the above configuration points to the service of the Data Index installed by the Orchestrator Operator.
```

Comment:
would it make sense to move this section before the yaml so the users know what to pay attention to as they prepare it?

Reply:
Makes sense to me - I moved it above. Let me know if that works